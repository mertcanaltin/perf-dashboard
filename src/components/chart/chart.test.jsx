import {render,screen} from '@testing-library/react'
import Chart from './chart';
test('Chart test', ()=>{
    const { debug,container } = render(<Chart/>);


    const titlePerf = container.querySelector('.title');

    expect(titlePerf).toBeInTheDocument();

})