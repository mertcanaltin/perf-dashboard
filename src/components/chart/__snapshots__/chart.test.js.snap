// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart test 1`] = `
<div
  class="site-card-wrapper"
>
  <div
    class="ant-card ant-card-bordered title"
  >
    <div
      class="ant-card-body"
    >
      <div
        class="recharts-responsive-container"
        style="width: 99%; height: 100%;"
      />
    </div>
  </div>
   
</div>
`;
